translations/en-us.json

```diff
@@ -14,9 +14,9 @@
   },
   "cache": {
     "no_data": "No data found for this address.",
-    "title": "MyStreet",
+    "title": "my-street",
     "bullet_one": "Click D.C. or L.A. navigation options above to use the application.",
-    "bullet_two": "This <a href=\"https://github.com/Esri/MyStreet\">project</a> is an <a href=\"http://emberjs.com/\">Ember App</a> that demonstrates the <a href=\"https://github.com/Esri/MyStreet/blob/master/hub-ready.md\">requirements of hub-ready</a> applications.",
+    "bullet_two": "This <a href=\"https://github.com/Esri/my-street\">project</a> is an <a href=\"http://emberjs.com/\">Ember App</a> that demonstrates the <a href=\"https://github.com/Esri/my-street/blob/master/hub-ready.md\">requirements of hub-ready</a> applications.",
```

Comment:
we will need to change the repo name to match this URL, right?

Will we also need to change the older repo (/ArcGIS/MyStreet) that currently redirects to here.

Reply:
> we will need to change the repo name to match this URL, right?

absolutely

> Will we also need to change the older repo (/ArcGIS/MyStreet) that currently redirects to here.

you mean (/ArcGIS/your-street)? my vote in https://github.com/ArcGIS/your-street/issues/11 is to just 🔪  that one.

Reply:
Fine to 🔪 , double checking with Andrew on that issue you posted. Also looks like there is one more issue hanging out there that could be brought over for posterity. 

What I mean is that if you navigate to `https://github.com/ArcGIS/MyStreet` you are auto-redirected to `Esri/MyStreet`. Not a big deal, but would be nice to have all roads end in one place.